lib/pipelines/compileSoy.js

```diff
@@ -108,17 +108,27 @@ function extractParams() {
 			var parsed = getParsedSoy(soyJsPath, file.contents);
 
 			templateData[soyJsPath] = {};
-			parsed.templates.forEach(function(cmd) {
-				if (cmd.deltemplate) {
+
+			parsed.body.forEach(function(template) {
+				if (template.deltemplate) {
 					return;
 				}
 
-				var templateName = cmd.name;
+				var templateName = template.id.name;
 				templateData[soyJsPath][templateName] = {params: [], types: {}};
-				cmd.params.forEach(function(tag) {
-					addTemplateParam(soyJsPath, templateName, tag.name);
-					templateData[soyJsPath][templateName].types[tag.name] = tag.type;
-				});
+
+				var extractParam = function(param) {
+					addTemplateParam(soyJsPath, templateName, param.name);
+					templateData[soyJsPath][templateName].types[param.name] = param.paramType;
+				}
+
+				// Parse inner params
+				template.params.forEach(extractParam);
```

Comment:
@jbalsas Nope, you should be good here: https://github.com/metal/soyparser/blob/master/src/types.ts#L217 👍